modules/building-memcached-operator-using-osdk.adoc

```diff
@@ -69,6 +70,60 @@ update the generated code for that resource type:
 $ operator-sdk generate k8s
 ----
 
+. *Optional: Add custom validation to your CRD.*
++
+OpenAPIv3 schemas are added to CRD manifests in the `spec.validation` block when
+the manifests are generated. This validation block allows Kubernetes to validate
+the properties in a Memcached CR when it is created or updated.
++
+Additionally, a `pkg/apis/<group>/<version>/zz_generated.openapi.go` file is
+generated containing the Go representation of this validation block if the
+`+k8s:openapi-gen=true annotation` is present above the `Kind` type declaration,
+which is present by default. This auto-generated code is your Go `Kind` type's
+OpenAPI model, from which you can create a full OpenAPI spec and generate a
```

Comment:
Consider spelling out and capitalizing spec (`OpenAPI Specification`) here (and abbreviating as `OAS` if used elsewhere). https://swagger.io/specification/